Common/Interfaces/IAlgorithm.cs

```diff
@@ -156,14 +156,6 @@ AlgorithmStatus Status
             set;
         }
 
-        /// <summary>
-        /// Gets a flag indicating whether or not this algorithm uses the QCAlgorithmFramework
-        /// </summary>
-        bool IsFrameworkAlgorithm
```

Comment:
Was this replaced with something else or do we no longer treat framework/non-framework differently? I recall we needed this for the UI to know if it was a framework algorithm -- IIRC, we also used this in the backend somewhere to decide to turn on/off the alpha handler (didn't see reference to that code in this PR -- might have codewebs in my head :P )

Reply:
Yes good point! We are working on the UI and backend changes too 👍